search/search_repository.go

```diff
@@ -346,6 +352,7 @@ var searchKeyMap = map[string]string{
 	"area":         workitem.SystemArea,
 	"iteration":    workitem.SystemIteration,
 	"assignee":     workitem.SystemAssignees,
+	"title":        workitem.SystemTitle,
```

Comment:
should there be some constants for these search keys ? are you supposed to use the same value as in https://github.com/fabric8-services/fabric8-wit/pull/1748/files#diff-175cd1756eb938ba265113030ff9eedf ?

Reply:
yes @xcoulon they are same and should be from the same source

Reply:
I will do it in a separate PR.

Reply:
It looks they are not exactly same. There is a `system.` prefix in one place.

Reply:
yes, they aren't the same as the ones in the `workitem` pkg, but they probably deserve their own constants. Ok for doing it in a separate PR if you prefer.